packages/server/lib/controllers/proxy/allProxy.ts

```diff
@@ -303,7 +303,7 @@ export function parseHeaders(req: Pick<Request, 'rawHeaders'>) {
     return forwardedHeaders;
 }
 
-async function handleResponse({ res, responseStream, logCtx }: { res: Response; responseStream: AxiosResponse; logCtx: LogContext }) {
+export async function handleResponse({ res, responseStream, logCtx }: { res: Response; responseStream: AxiosResponse; logCtx: LogContext }) {
```

Comment:
not ideal to export this private logic just to be able to unit test it but the alternative  (setting up all the machinery in the tests or refactoring/extracting this logic into its own file) is also adding complexity